packages/plugin-ga-events-forwarder-browser/src/ga-events-forwarder.ts

```diff
@@ -0,0 +1,139 @@
+import { getGlobalScope } from '@amplitude/analytics-client-common';
+import { BaseEvent, BrowserClient, BrowserConfig, EnrichmentPlugin, Logger } from '@amplitude/analytics-types';
+import { GA_PAYLOAD_HOSTNAME_VALUE, GA_PAYLOAD_PATHNAME_VALUE } from './constants';
+import { isTrackingIdAllowed, isVersionSupported, parseGA4Events, transformToAmplitudeEvents } from './helpers';
+
+type BrowserEnrichmentPlugin = EnrichmentPlugin<BrowserClient, BrowserConfig>;
+interface Options {
+  trackingIds?: string | string[];
+}
+
+/**
+ * Returns an instance of `gaEventsForwarderPlugin`. Add this plugin to listen for events sent to Google Analytics,
+ * transform the events and send the events to Amplitude.
+ *
+ * ```html
+ * <script>
+ *   amplitude.add(gaEventsForwarder.plugin());
```

Comment:
is aEventsForwarder.plugin() right?

Reply:
yes, this is how it looks like for script installation